scripts/ci/images/build-el8.sh

```diff
@@ -1,5 +1,4 @@
-
-#~/bin/bash
+#!/bin/bash
```

Comment:
Whoops, obviously I made a typo, and the fact that I had shellcheck ignore it, allowed it to sneak through.  Thanks for fixing.